Gems/PhysX/Code/Source/EditorRigidBodyComponent.cpp

```diff
@@ -132,7 +132,7 @@ namespace PhysX
         if (serializeContext)
         {
             serializeContext->Class<EditorRigidBodyConfiguration, AzPhysics::RigidBodyConfiguration>()
-                ->Version(2, &ClassConverters::EditorRigidBodyConfigVersionConverter)
+                ->Version(2)
```

Comment:
question: In the other case where we removed the version converted the number was dropped by 1 but is the same here. Is it worth being consistent and either keeping it the same or dropping by one (or incrementing?)

Reply:
Changed the version to 1, thanks!